ocs_ci/ocs/pgsql.py

```diff
@@ -216,14 +251,20 @@ def wait_for_pgbench_status(self, status, timeout=None):
             timeout (int): Time in seconds to wait
 
         """
-        timeout = timeout if timeout else 1200
+        timeout = timeout if timeout else 1800
```

Comment:
@OdedViner could you please log why we need such a big timeout?

Reply:
sometimes  with the default values in the benchmark yaml  the pgbench pod is not getting completed within the specified time and the tests are failing. 
I think it is varying with the infrastructure.
So, for now we set the timeout to 30mins and will start monitoring  each pg bench pods for each run. 
Based on the results we will define the timeout again